appserver/admin/gf_template/src/main/resources/config/domain.xml

```diff
@@ -261,6 +257,7 @@
         <jvm-options>[1.8.0u121|1.8.0u160]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.7.jar</jvm-options>
         <jvm-options>[1.8.0u161|1.8.0u190]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.8.jar</jvm-options>
         <jvm-options>[1.8.0u191|1.8.0u500]-Xbootclasspath/p:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap-1.8.1.jar</jvm-options>
+        <jvm-options>[9|]-Xbootclasspath/a:${com.sun.aas.installRoot}/lib/grizzly-npn-bootstrap.jar</jvm-options>
```

Comment:
Why does JDK 11 need the npn-boot jar? Doesn't JDK 11 support NPN negotiation?

Reply:
[Http2AddOn](https://github.com/eclipse-ee4j/grizzly/blob/master/modules/http2/src/main/java/org/glassfish/grizzly/http2/Http2AddOn.java#L121) requires `grizzly-npn-bootstrap.jar#org.glassfish.grizzly.npn.AlpnServerNegotiator` class.

Reply:
OK that may need Grizzly work then as Alpn negotiation I thought was in the SSL stack of the JDK now

Reply:
probably not enough to stop this PR going in but can you raise an issue on Grizzly

Reply:
Added the `grizzly-npn-api.jar` to classpath for JDK 9+